website/www/site/content/en/contribute/_index.md

```diff
@@ -50,53 +50,106 @@ There are lots of opportunities to contribute. You can for example:
    Cloud Dataflow, etc)
  - improve or add IO connectors
  - add new transform libraries (statistics, ML, image processing, etc)
- - work on the core programming model (what is a Beam pipeline and how does it
-   run?)
+ - work on the core programming model (read more about what a Beam pipeline is and how it runs here in [Documentation](https://beam.apache.org/documentation/basics/))
  - improve the developer experience (for example, Windows guides)
  - add answers to the [contribution FAQ](
  https://cwiki.apache.org/confluence/display/BEAM/Contributor+FAQ)
  - organize local meetups of users or contributors to Apache Beam
 
+Below is a tutorial for contributing code to Beam, covering our tools and typical process in
+detail.
+
 </div>
 
-<a class="arrow-list-header" data-toggle="collapse" href="#collapseContributing" role="button" aria-expanded="false" aria-controls="collapseContributing">
-   {{< figure src="/images/arrow-icon_list.svg">}}
+## Get Started
 
-## Contributing code
+### Connect with the Beam community
 
-  </a>
+1. Consider [subscribing to the dev@beam.apache.org]( https://beam.apache.org/community/contact-us/) mailing list, especially if you plan to make more than one change or the change will be large. All decisions are consensus-based and happen on the public mailing list.
```

Comment:
Extra space here:  "( https:". I'm pretty sure the Markdown parser will ignore that, but ... FYI.

Reply:
changed